org.unitystation.StationHub.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: org.unitystation.StationHub
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: stationhub.sh
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet
+finish-args:
+  - --socket=x11
```

Comment:
if it really works natively on Wayland, please change it to `--socket=fallback-x11`.

Reply:
I haven't tested in Wayland at all but I am pretty sure it does not run natively. Can it run in XWayland without the Wayland permission?